recipe/meta.yaml

```diff
@@ -17,40 +16,60 @@ build:
 
 requirements:
   host:
-    - python >=3.6,<3.10
-    - poetry
+    - python >=3.6.2,<3.11
+    - poetry-core >=1.0.0
     - pip
   run:
-    - python >=3.6,<3.10
-    - boto3 >=1.16.8,<2.0.0
-    - botocore >=1.19.8,<2.0.0
-    - numpy >=1.18.0,<2.0.0
-    - pandas >=1.1.0,<2.0.0
-    - pyarrow >=2.0.0,<5.1.0
-    - redshift_connector ~=2.0.887
+    - boto3 >=1.20.17,<2.0.0
+    - botocore >=1.23.17,<2.0.0
+    - jsonpath-ng >=1.5.3,<2.0.0
+    - numpy >=1.18.0,<1.22
+    # Conda should choose the correct numpy versions for the different python versions.
+    - openpyxl >=3.0.0,<3.1.0
+    - opensearch-py >=1.0.0,<2.0.0
+    - pandas >=1.1.0,<1.4.0
+    # Conda should choose the correct pandas versions for the different python versions.
+    - pg8000 >=1.16.0,<1.23.0
+    - progressbar2 >=3.53.3,<4.0.0
+    - pyarrow >=2.0.0,<6.1.0
     - pymysql >=0.9.0,<1.1.0
-    - pg8000 >=1.16.0,<1.22.0
-    - openpyxl ~=3.0.0
+    - python >=3.6.2,<3.11
+    - redshift_connector >=2.0.889,<2.1.0
+    - requests-aws4auth >=1.1.1,<2.0.0
+    - xlrd >=2.0.1,<3.0.0
+    - xlwt >=1.3.0,<2.0.0
+    # Both are only needed for python3.6 but have been included to keep the package noarch.
+  run_constrained:
+    - pyodbc >=4.0.32,=4.0
 
 test:
   imports:
     - awswrangler
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
-  home: https://github.com/awslabs/aws-data-wrangler
-  license: Apache-2.0
-  license_family: Apache
-  license_file: LICENSE.txt
```

Comment:
This is not super important, I just used the sorting that is normally used with conda packages so it's easier to read. Feel free to reject that.